src/components/tooltip/index.tsx

```diff
@@ -41,7 +41,12 @@ const Tooltip: FC<Props> = ({
     <div className="inline-block" onClick={(e) => e.preventDefault()}>
       <TooltipTrigger
         content={
-          <div className="rounded-4 shadow-hard p-15 bg-white">
+          <div
+            className="rounded-4 shadow-hard p-15 bg-white"
+            onTouchEnd={(e) => {
```

Comment:
Do we understand why we need to do that onTouchEnd?

It concerns me a bit because the support is very bad on Safari. I quickly checked it on an iPhone 8 and the fix does not work. It is only officially supported after Safari 14.6 which is the newest version

https://caniuse.com/mdn-api_element_touchend_event

Reply:
@talamcol interesting. Well, this was a 'quick fix' that we added everytime we consumed the Tooltip component. So, the purpose of this PR was to centrealize it so we don't need to remember adding this everytime.

Reply:
@gzimh I think another way to solve this issue is to apply `pointer-event: all` to the wrapping `div`. The problem here is that the tooltip is absolute positioned and this should solve the issue.

Browser support looks better for this. But it's up to you, maybe let's hear another opinion

Reply:
Sorry for the confusion, it does only work the other way around. You can ignore my comment :)

Reply:
I agree with the sentiment though. let's try to understand what the initial fix does and why it was added exactly. is it about the tooltip being rendered on a card in a carousel? is there something about those touch/swipe events bubbling? I'd like to understand the underlying issue and solution better before we introduce it globally

one line above we also have a `e.preventDefault` on the click event. this is unusual as there's rarely a reason to prevent bubbling. can you dig a bit deeper before we settle for this workaround?

Reply:
would you like any support here?

Reply:
@masone i have some time today to dig a bit deeper into this. But i dont see a way other than stoping event propagation because the click/touch event will reach the Listing item and that will trigger it to redirect (just as an example, same thing would happen elsewhere). Maybe im missing something, will look into this a bit more

Reply:
I remember only the ontouch worked, but the click didn't that sounds weird!

Reply:
let's spend an hour and then proceed as-is. at least, it's tucked away nicely now

Reply:
On desktop it was working as it should (click). The onTouchEnd prevention is to fix the issue on mobile only